cmd/edenStart.go

```diff
@@ -0,0 +1,76 @@
+package cmd
+
+import (
+	"github.com/lf-edge/eden/pkg/utils"
+	"github.com/spf13/cobra"
+	"log"
+	"os"
+	"path"
+	"path/filepath"
+	"runtime"
+)
+
+var (
+	adamDist         string
+	adamPort         string
+	adamForce        bool
+	eserverImageDist string
+	eserverPort      string
+	eserverPidFile   string
+	eserverLogFile   string
+	evePidFile       string
+	eveLogFile       string
+)
+
+var startCmd = &cobra.Command{
+	Use:   "start",
+	Short: "start harness",
+	Long:  `Start harness.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		adamPath, err := filepath.Abs(adamDist)
+		if err != nil {
+			log.Fatalf("adam-dist problems: %s", err)
+		}
+		command, err := os.Executable()
+		if err != nil {
+			log.Fatalf("cannot obtain executable path: %s", err)
+		}
+		log.Printf("Executable path: %s", command)
```

Comment:
Can we please start using https://github.com/sirupsen/logrus so that we get clearly defined logging levels (like this one right here will become INFO) and hook it up to Cobra flags/Viper configuration a'la https://le-gall.bzh/post/go/integrating-logrus-with-cobra/

Reply:
Done